build-system/global-configs/canary-config.json

```diff
@@ -5,6 +5,6 @@
   "amp-live-list": 1,
   "amp-ios-overflow-x": 1,
   "amp-experiment": 1,
-  "expAdsenseA4A": 0.05,
-  "expDoubleclickA4A": 0.05
+  "expAdsenseA4A": 1.0,
```

Comment:
Thanks!  FWIW, I did get a new PR in place: #4847 Feel free to go ahead with that one instead, if Travis is happy in time.